### PR TITLE
fix: enable ts check

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

**Review by doodlewind — APPROVED** [review]
LGTM

**Comment by himself65** [comment]
I found bugs in the new changes. We need a test for this part

File: packages/blocks/src/code-block/components/syntax-code-block.ts [hunk 1]
```diff
@@ -102,15 +105,15 @@ class Syntax extends Module {
 
   constructor(quill: Quill, options: SyntaxCodeBlockOptions) {
     super(quill, options);
-    this.language = options.language;
+    this.language = options.lang;
```
Comment (thorseraq) [inline packages/blocks/src/code-block/components/syntax-code-block.ts:108]: Thanks! Now it corresponds to  type of `SyntaxCodeBlockOptions`, I think https://github.com/toeverything/blocksuite/blob/d1b177466e5468a238f72ed0a1d38140af4b3f0f/packages/blocks/src/code-block/code-block.ts#L110 need to be changed to `lang` as well.

**Comment by himself65** [comment]
Due to quill doesn't have type declaration. So we use `any script`

**Comment by doodlewind** [comment]
> Due to quill doesn't have type declaration. So we use `any script`

I think we have `@types/quill`

**Comment by himself65** [comment]
> > Due to quill doesn't have type declaration. So we use `any script`
> 
> I think we have `@types/quill`

It's not fully. for example it doesn't have the type under `core/*`

**Pull request merged**